src/utils/logging.rs

```diff
@@ -5,7 +5,7 @@ use std::{
     path::PathBuf,
 };
 
-use crate::utils::consts::{MOSAIC_TMP_LOG_DIR, MOSAIC_TMP_LOG_FILE};
+use crate::utils::consts::*;
```

Comment:
I actually very much prefer to be explicit about imports. I think it makes it much clearer when you're trying to figure out where something is coming from.